getting-started/vs-code-integration/dpl-wizard.md

```diff
@@ -0,0 +1,51 @@
+
+# Configuring Document Processing Libraries
+
+This article demonstrates how to add a reference to the [Telerik Document Processing Libraries (DPL)](https://www.telerik.com/document-processing-libraries) packages in an existing Telerik UI for Blazor application using the Document Processing Libraries Wizard. The wizard is included in the [Telerik UI for Blazor Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=TelerikInc.blazortemplatewizard) and enables you to easily add the required packages when working with the Telerik Document Processing Library.
+
+## Getting the Wizard
+
+To use the Document Processing Libraries Wizard, install the [Telerik UI for Blazor Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=TelerikInc.blazortemplatewizard).
+
+## Starting the Wizard
+
+`Using the File Explorer Context Menu`
+
+To use **Document Processing Libraries Configuration Wizard** from the **VS Code**'s file explorer's context menu:
+ - Right-click on a file in the files tree.
+ - Select the `Telerik UI for Blazor: Add Document Processing Libraries`.
+
+  ![Configure Document Processing Wizard, Context Menu](images/DPLBlazor_ContextMenu.png)
+
+`Using the Code Editor Command Palette`
+
+To use **Document Processing Libraries Configuration Wizard** from the **VS Code** command palette:
+
+ - Open a `.razor` file or a `Razor` language in the code editor.
+ - Select the `Telerik UI for Blazor: Add Document Processing Libraries`.
+
+ ![Configure Document Processing Wizard, Command Palette](images/DPLBlazor_Pallete.png)
+
+After executing one of the options the wizard must appear like this: 
+
+ ![Configure Document Processing Wizard](images/DPLWizardBlazor.png "Configure Document Processing Wizard")
+
+ The Telerik Document Processing Configuration wizard allows you to select which libraries your project would use. Also have options to use `Dev` or `Trial` license and selection of the `.csproj` file, in which you want the installation to be provded. This option is useful in cases, when the user have many opened projects in the workspace. If a library you selected has any dependencies, they are automatically checked. In the table below there is a list of the assemblies that will be selected for each library:
+
+|Library  |Packages |
+|---------|---------|
+|RadPdfProcessing    |Documents.Core<br>Documents.Fixed<br>Zip<br>Documents.ImageUtils<br>Documents.Fixed.FormatProviders.Image.Skia<br>Documents.CMapUtils         |
+|RadSpreadProcessing     |    Documents.Core<br>Documents.Fixed<br>Documents.Spreadsheet<br>Documents.SpreadSheet.FormatProviders.OpenXml<br>Documents.SpreadSheet.FormatProviders.Pdf<br>Documents.Spreadsheet.FormatProviders.Xls<br>Documents.ImageUtils<br>Zip     |
+|RadWordsProcessing     |   Documents.Core<br>Documents.Fixed<br>Documents.Flow<br>Documents.Flow.FormatProviders.Pdf<br>Documents.ImageUtils<br>Documents.Flow.FormatProviders.Doc<br>Documents.DrawingML<br>Zip      |
```

Comment:
```suggestion
| Library | Packages |
| --- | --- |
| RadPdfProcessing | Documents.Core <br /> Documents.Fixed <br /> Zip <br /> Documents.ImageUtils <br /> Documents.Fixed.FormatProviders.Image.Skia <br /> Documents.CMapUtils |
| RadSpreadProcessing | Documents.Core <br /> Documents.Fixed <br /> Documents.Spreadsheet <br /> Documents.SpreadSheet.FormatProviders.OpenXml <br /> Documents.SpreadSheet.FormatProviders.Pdf <br /> Documents.Spreadsheet.FormatProviders.Xls <br /> Documents.ImageUtils <br /> Zip |
| RadWordsProcessing | Documents.Core <br /> Documents.Fixed <br /> Documents.Flow <br /> Documents.Flow.FormatProviders.Pdf <br /> Documents.ImageUtils <br /> Documents.Flow.FormatProviders.Doc <br /> Documents.DrawingML <br /> Zip |
```

Reply:
How aware is the user of the RadPdfProcessing name? It's not really visible in the wizard. Also, the ordering of the libraries in the table does not correspond to the order of choice in the wizard. You might want to retain the order and the naming from the wizard. If you need to mention `RadPdfProcessing`, do it in a separate column. For example, your columns could be `| Wizard Option | Library Name | Packages |` (or something else that makes more sense).

Reply:
We need to assume that when a developer is adding the DPL packages, they must already know what DPL is and what it consists of. Nevertheless, I added links to the different library Overview pages.